qiskit_ibm_provider/transpiler/passes/scheduling/scheduler.py

```diff
@@ -129,8 +138,18 @@ def _visit_conditional_node(self, node: DAGNode) -> None:
         """
         # Special processing required to resolve conditional scheduling dependencies
         if node.op.condition_bits:
```

Comment:
Do we still need this if block? With change at [this line](https://github.com/Qiskit/qiskit-ibm-provider/pull/391/files#diff-42c9f6575925d139f032253a66a5610f6aa3d814b6fed1497258cbaae8a55762R109) the `_visit_conditional_node` will be never called when conditional bits are empty, so this is obvious.